packages/node/jest.config.js

```diff
@@ -32,6 +32,6 @@ module.exports = {
   "transform": {
     "^.+\\.tsx?$": "ts-jest"
   },
-  "verbose": true,
+  "verbose": false,
```

Comment:
I think this suppresses console.log or something...

Reply:
I _think_ jest behavior has changed since then. I had `verbose` set to true and logs weren't showing up unless `sleep`ed surrounding the relevant code. Putting verbose to `false` displayed the logs without needing to sleep.